src/testpool.jl

```diff
@@ -126,21 +126,31 @@ function provision_all_test_engines()
 end
 
 """
-    resize_test_engine_pool(5)
-    resize_test_engine_pool(5, get_next_engine_name)
+    resize_test_engine_pool(size::Int64, generator::Option{Function})
```

Comment:
```suggestion
    resize_test_engine_pool(size::Int64, generator::Option{Function}=nothing)
```

Reply:
Oops. Thanks! I lost nothing in a rewrite.